external/pom.xml

```diff
@@ -20,17 +20,13 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
+    <groupId>org.apache.kylin</groupId>
+    <version>3.1.0</version>
```

Comment:
not snapshot version? can it be ${project.version}?

Reply:
Oh, I am sorry, let me try to explain the problem I faced:

As I am the release manager of kylin 3.1.0, I am trying to use following command to do the release.
```sh
mvn release:prepare
```

And this plugin will check a lot of pre-requirements, at the end, it will throw an exception:
```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-release-plugin:2.5.3:prepare (default-cli) on project kylin: Can't release project due to non released dependencies :
[ERROR]     org.apache.kylin:kylin-shaded-guava:jar:3.1.0-SNAPSHOT:compile
[ERROR] in project 'Apache Kylin - Core Common' (org.apache.kylin:kylin-core-common:jar:3.1.0-SNAPSHOT)
[ERROR] -> [Help 1]
```

It means that a dependency which has a "SNAPSHOT" version and source code is provided is not forbidden.
Unluckily, the "shaded-guava" is a "SNAPSHOT" dependency, and it source code is not provided.

As a workaround, before this `mvn release:prepare` , I will install "shaded-guava" locally and then execute release command.